torch_xla/distributed/parallel_loader.py

```diff
@@ -40,7 +40,22 @@ def next(self):
 
 
 class ParallelLoader(object):
-  """ParallelLoader (PLACEHOLDER)"""
+  """Wraps an existing PyTorch DataLoader with background data upload.
+
+  Args:
+    loader: The PyTorch DataLoader to be wrapped.
+    devices: The list of devices where the data has to be sent. The i-th sample
```

Comment:
+1 to Ailing's suggestion about specifying variable types. For example, I think `devices` are a bit tough in this API without types. They appear sometimes as objects (like in `xla_device`), sometimes as device IDs (like in `DataParallel`), sometimes are strings (like in `get_xla_supported_devices`). I think it's fine that they appear in different ways but it's best to be more explicit in the docstring about the variable type

Reply:
Yeah ... IMHO we should standardize on `torch.device` for this, but that requires changes on all the test models, so will be matter for a following PR.